src/commands/meme.ts

```diff
@@ -0,0 +1,378 @@
+import Canvas from "@napi-rs/canvas";
+import {
+	ApplicationCommandOptionType,
+	ApplicationCommandType,
+	AttachmentBuilder,
+} from "discord.js";
+import { join } from "node:path";
+import { cwd } from "node:process";
+import { CustomClient, createCommand, normalizeError, sendError } from "../util";
+
+type Coord = [number, number];
+type TextPoint = {
+	coord: Coord;
+	dimensions: Coord;
+	color?: string;
+	rotate?: number;
+	fontSize?: number;
+	fontFamily?: string;
+	bold?: boolean;
+	italic?: boolean;
+	borderColor?: string;
+	text?: number;
+};
+
+const memes: Record<string, { dimensions: Coord; textPoints: TextPoint[] } | undefined> = {
+	"Drake Hotline Bling": {
+		dimensions: [1200, 1200],
+		textPoints: [
+			{ coord: [903, 300], dimensions: [596, 596] },
+			{ coord: [903, 900], dimensions: [596, 596] },
+		],
+	},
+	"Two Buttons": {
+		dimensions: [600, 908],
+		textPoints: [
+			{ coord: [160, 140], dimensions: [150, 83], rotate: -Math.PI / 18 },
+			{ coord: [350, 100], dimensions: [110, 70], rotate: -Math.PI / 18 },
+			{
+				coord: [308, 807],
+				dimensions: [546, 113],
+				fontFamily: "Impact",
+				color: "#ffffff",
+				borderColor: "black",
```

Comment:
Can't you be consistent with all colours and put the hex code in it? Also, considering the amount of times you've put those, I'd rather put the colours inside of a config file as they're being used constantly.

Reply:
You're right about the hex code, I'll fix it immediately. Fair enough about the config file but for the moment I won't do it; probably I'll do it when there are more templates.
Thanks for the feedback!